abb_irb6600_support/urdf/irb6600_225_255_macro.xacro

```diff
@@ -0,0 +1,348 @@
+        </geometry>
+        <xacro:material_abb_black />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_4_abb.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+    </link>
+    <link name="${prefix}link_4_abb2">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/visual/link_4_abb2.stl" />
+        </geometry>
+        <xacro:material_abb_black />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_4_abb2.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+    </link>
+    <link name="${prefix}link_5">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/visual/link_5.stl" />
+        </geometry>
+        <xacro:material_abb_orange />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_5.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+    </link>
+    <link name="${prefix}link_6">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/visual/link_6.stl" />
+        </geometry>
+        <xacro:material_abb_black />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_6.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+    </link>
+    <link name="${prefix}tool0" />
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint name="${prefix}joint_1" type="revolute">
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <parent link="${prefix}base_link" />
+      <child link="${prefix}link_1" />
+      <axis xyz="0 0 1" />
+      <limit lower="-3.14159265" upper="3.14159265" effort="0" velocity="1.74532925" />
+    </joint>
+    <joint name="${prefix}joint_1_mtr" type="fixed">
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <parent link="${prefix}link_1" />
+      <child link="${prefix}link_1_mtr" />
+    </joint>
+    <joint name="${prefix}joint_1_mtr2" type="fixed">
+      <origin xyz="0 0 0" rpy="0 0 0" />
+      <parent link="${prefix}link_1" />
+      <child link="${prefix}link_1_mtr2" />
+    </joint>
+    <joint name="${prefix}joint_1_wt" type="fixed">
```

Comment:
Cosmetic only: parallel bar structures can be modelled using [mimic joints](http://wiki.ros.org/urdf/XML/joint). Might be nice to do that here as well. I haven't investigated how that interacts with MoveIt though, so that might be something to test.

See the [M-900iA/260L](https://github.com/ros-industrial/fanuc_experimental/blob/3ef27dbf1402ee7e765479a6a48b1de530cec06b/fanuc_m900ia_support/urdf/m900ia260l_macro.xacro#L195-L208) for an example.